assets/post-connections.js

```diff
@@ -244,12 +244,89 @@ function togglePanels() {
   })
 }
 
+function clearSearch() {
+  document.getElementById('search').value = '';
```

Comment:
You'll want to call `searchData` here to refresh the list with the unfiltered list items again.